proto/trip.proto

```diff
@@ -304,30 +305,11 @@ message TripLeg {
     repeated uint32 speed = 3 [packed=true]; // decimeters per sec
   }
 
-  message Incident {
-    enum Type {
-      NOT_SET = 0;
-      ACCIDENT = 1;
-      CONGESTION = 2;
-      CONSTRUCTION = 3;
-      DISABLED_VEHICLE = 4;
-      LANE_RESTRICTION = 5;
-      MASS_TRANSIT = 6;
-      MISCELLANEOUS = 7;
-      OTHER_NEWS = 8;
-      PLANNED_EVENT = 9;
-      ROAD_CLOSURE = 10;
-      ROAD_HAZARD = 11;
-      WEATHER = 12;
-    };
-    optional Type type = 1;
-    optional string description = 2;
-    optional uint64 start_time = 3;     // START_TIME as seconds since epoch UTC
-    optional uint64 end_time = 4;       // END_TIME seconds since epoch UTC
-    optional uint64 creation_time = 5;  // time the incident was created/updated in seconds since epoch UTC
-    optional uint64 id = 6;             // An id that can be used for joining incidents in route with map
-    optional uint32 begin_shape_index = 7;
-    optional uint32 end_shape_index = 8;
+  message ValhallaIncident {
+    optional valhalla_sideloaded.Incident metadata = 1;
```

Comment:
how does this work!? mixing proto2 and proto3? crazy stuff. also instead of doing this...

could we add a metadata field to the edgetoincident structure, but only fill it out when the incident is fetched in the graphreader. then we could just copy the incident verbatim. i mean maybe the secondary look up for the metadata is also overkill im not sure. just some ideas to simplify this